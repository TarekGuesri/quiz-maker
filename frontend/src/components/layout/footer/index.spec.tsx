import React from "react";
import { cleanup, render } from "@testing-library/react";
import { Footer } from ".";

describe("src/components/layout/navbar", () => {
  afterEach(() => cleanup());

  test("Render Fotter component", () => {
    const renderedComponent = render(<Footer />);

    expect(renderedComponent.container).toMatchSnapshot();
  });
});
