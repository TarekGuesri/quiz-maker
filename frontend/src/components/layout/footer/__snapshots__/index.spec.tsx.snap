// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/layout/footer Render Footer component 1`] = `
<div>
  <div
    class="MuiBox-root css-1swgbew"
  >
    <div
      class="MuiTypography-root MuiTypography-body1 css-1oc1ahk-MuiTypography-root"
    >
      Â© 
      2022
       TARIQ LAMIN GUESRI. ALL RIGHTS RESERVED.
    </div>
  </div>
</div>
`;
